src/js/clickable-component.js

```diff
@@ -10,15 +10,24 @@ import document from 'global/document';
 import assign from 'object.assign';
 
 /**
- * Clickable Component which is clickable or keyboard actionable, but is not a native HTML button
+ * Clickable Component which is clickable or keyboard actionable,
+ * but is not a native HTML button.
  *
- * @param {Object} player  Main Player
- * @param {Object=} options Object of option names and values
  * @extends Component
- * @class ClickableComponent
  */
 class ClickableComponent extends Component {
 
+  /**
+   * constructor for a ClickableComponent
+   * but is not a native HTML button.
```

Comment:
These look like they should be on the same line. And "constructor" should be capitalized. And there should be a comma after "ClickableComponent".